### PR TITLE
add support for alerta.io 

### PR DESCRIPTION
**Pull request by kattunga**
Added suport for alerta.io
screenshot at 2017-12-20 15 09 44

  fixes: #3147

### CODE REVIEW

**Comment by ktsaou** [comment]
nice! give me a day to review this properly.

File: conf.d/health_alarm_notify.conf [hunk 1]
```diff
@@ -275,6 +278,26 @@ SLACK_WEBHOOK_URL=""
 DEFAULT_RECIPIENT_SLACK=""
 
 
+#------------------------------------------------------------------------------
+# alerta (alerta.io) global notification options
+
+# multiple recipients can be given like this:
+#                  "Environment1 Environment1 ..."
```
Comment (ktsaou) [inline conf.d/health_alarm_notify.conf]: Envirorment1 Environment2 ...

File: conf.d/health_alarm_notify.conf [hunk 2]
```diff
@@ -275,6 +278,26 @@ SLACK_WEBHOOK_URL=""
...
+
+# enable/disable sending alerta notifications
+SEND_ALERTA="YES"
+
+# set your alerta server API url
```
Comment (ktsaou) [inline conf.d/health_alarm_notify.conf]: Can you add some information for the place this API URL can be obtained from?
Is the API URL the same for all users? You could add it here if it is.

File: conf.d/health_alarm_notify.conf [hunk 3]
```diff
@@ -275,6 +278,26 @@ SLACK_WEBHOOK_URL=""
...
+ALERTA_WEBHOOK_URL=""
+
+# Login to you arerta server and create an API KEY with write permisions
+ALERTA_API_KEY=""
+
+# if a role's recipients are not configured, a notification will be send to
+# this Environment (empty = do not send a notification for unconfigured roles):
+DEFAULT_RECIPIENT_ALERTA=""
+
+
```
Comment (ktsaou) [inline conf.d/health_alarm_notify.conf]: At the end of this file you need to add entries like these:

```
role_recipients_alerta[sysadmin]="${DEFAULT_RECIPIENT_ALERTA}"
```

These are used to support role based recipients.

**Comment by ktsaou** [comment]
ok, it took me more than a day... lot of requests.
Well, you have done an excellent job. I made just a few minor comments.

You will have to also add a wiki page for alerta, here: 

screenshot from 2017-12-27 01-19-20

The idea is to help other users willing to setup alerta.io for their netdata. So, a few comments of your experience, a few screenshots of the expected result and some directions to set it up would be nice.

**Comment by kattunga** [comment]
Yes of course, right now i am out of my country with limited access to internet.
Next week i can upload an entry in the wiki to setup an alerta server and configure it to work with netdata.

**Comment by ktsaou** [comment]
perfect. Post here when you are ready.

**Comment by kattunga** [comment]
check this please.

https://github.com/firehol/netdata/wiki/Alerta-monitoring-system

image

I will improve it over the following days.
Feedback will be appreciated.

Regards

**Comment by ktsaou** [comment]
seems good.
So, you have just a few minor edits in `health_alarm_notify.conf` and we are good to merge!
excellent job!

**Comment by kattunga** [comment]
I think its ok now

**Comment by ktsaou** [comment]
merged!
thank you!

**Comment by ktsaou** [comment]
alerta

logo for the infographic.

**Comment by kattunga** [comment]
There was a mistake. I submit PR with a little fix.

**Pull request merged**